neonUtilities/R/zipsByProduct.R

```diff
@@ -217,19 +216,30 @@ zipsByProduct <- function(dpID, site="all", startdate=NA, enddate=NA, package="b
         t <- tryCatch(
           {
             suppressWarnings(downloader::download(zip.urls$URL[j], zip_out,
-                                 mode="wb", quiet=T))
+                                                  mode="wb", quiet=T))
           }, error = function(e) { e } )
 
         if(inherits(t, "error")) {
-          writeLines("File could not be downloaded. URLs may have expired. Trying new URLs.")
-          zip.urls <- getZipUrls(tmp.files, month.urls=month.urls, avg=avg, package=package,
-                                 max.pub=max.pub, max.pub.site=max.pub.site, dpID=dpID, messages=messages) %>%
-            tidyr::drop_na()
+          writeLines(paste0(zip.urls$name[j], " could not be downloaded. URLs may have expired. Trying new URLs."))
+
+          quiet <- function(x) {
+            sink(tempfile())
+            on.exit(sink())
+            invisible(force(x))
+          }
```

Comment:
@cklunch Can we leave this here or does it need a stand alone script?

Reply:
@NateMietk Let's put it in its own script. Even though it's tiny, it will be smoother to have it set up that way. I think that's the last change needed!